drivers/entropy/entropy_stm32.c

```diff
@@ -492,6 +495,8 @@ static int entropy_stm32_rng_init(const struct device *dev)
 	LL_RNG_SetHealthConfig(dev_data->rng, DT_INST_PROP(0, health-test-config));
 #endif
 
+	pm_constraint_set(PM_STATE_SUSPEND_TO_IDLE);
```

Comment:
This one totally prevents the use of PM, right ?

Reply:
yes, for a small amount of time. At startup the rng pool is filled, so the clock needs to be on for that time. It's stopped when the rng pool is filled (in the isr)

Reply:
Should I add a comment to explain the flow?

Reply:
It would be helpful, txs